install_config/registry/extended_registry_configuration.adoc

```diff
@@ -80,42 +80,58 @@ We recommend against rebooting the entire cluster because that incurs
 unnecessary downtime for pods and does not actually clear the cache.
 
 [[whitelisting-docker-registries]]
-== Whitelisting Docker Registries
-
-You can specify a whitelist of docker registries, allowing you to curate a set
-of images and templates that are available for download by {product-title}
-users. This curated set can be placed in one or more docker registries, and then
-added to the whitelist. When using a whitelist, only the specified registries
-are accessible within {product-title}, and all other registries are denied
-access by default.
+== Configuring an External Registry Search List
+
+You can use the *_/etc/containers/registries.conf_* file to create a list of 
+Docker registries to search for container images. 
+
+The *_/etc/containers/registries.conf_* file is a list of registry servers 
+that {product-title} should search against when a user pulls an image using
+the image short name. For example, if the registry search list contains 
+`registry.redhat.io`, `docker.io`, and `registry.example.com`, and the 
```

Comment:
It might help to list the list;

- `registry.redhat.io`
- `docker.io`
- `registry.example.com`

> For example, if the previous list is searched for `myimage:lastest`, OCP searches from first to last until it finds a match or no match is found.

Or some such; but maybe that's excessive and it's clear enough as written.